backend/sidedish/src/main/java/codsquad/team17/sidedish/dto/ItemDto.java

```diff
@@ -0,0 +1,89 @@
+package codsquad.team17.sidedish.dto;
+
+import codsquad.team17.sidedish.domain.Image;
+import codsquad.team17.sidedish.domain.Item;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@JsonPropertyOrder({"item_id", "title", "description", "n_price", "s_price",
+        "badge", "image", "delivery_type", "detail_url"})
+public class ItemDto {
+
+    private static final String URL = "http://ec2-15-164-123-251.ap-northeast-2.compute.amazonaws.com:8080";
+
+    @JsonProperty("item_id")
+    private final Long itemId;
+
+    private final String title;
+
+    private final String description;
+
+    @JsonProperty("n_price")
+    private final int normalPrice;
+
+    @JsonProperty("s_price")
+    private final int salePrice;
+
+    @JsonProperty("delivery_type")
+    private List<String> deliveryType = Arrays.asList(new String[]{"새벽배송", "전국택배"});
```

Comment:
꼭 새로운 배열을 선언할 필요는 없습니다. [자바독 참고](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Arrays.html#asList(T...))

```suggestion
    private List<String> deliveryType = Arrays.asList("새벽배송", "전국택배);

Reply:
오.. 감사합니다